CHANGELOG.md

```diff
@@ -1,8 +1,48 @@
+### citus v5.1.0 (May 17, 2016) ###
+
+* Adds support for using EXPLAIN on distributed queries
+
+* Improves planner performance when using tables with many shards
+
+* Adds ability to pass columns as arguments to function in UPDATEs
+
+* Introduces transaction manager for use by multi-shard commands
+
+* Adds distributed COPY to rapidly populate distributed tables
+
+* Improves user experience during DROP EXTENSION commands
+
+* Increases INSERT throughput via shard pruning optimizations
+
+* Chooses router executor dynamically
```

Comment:
Rename to something like "recognize fast-path SELECT statements dynamically instead of via configuration"? i.e. more value based, rather than using 'router' executor terminology.